cylc/flow/cfgspec/globalcfg.py

```diff
@@ -94,9 +94,9 @@
         The number of old run directory trees to retain at start-up.
     ''')
 
-    # suite
-    with Conf('cylc', desc='''
-        Default values for entries in flow.cylc ``[cylc]`` section.
+    with Conf('scheduler', desc='''
+        Default values for entries in flow.cylc ``[scheduler]`` section.
+        This should not be confused with scheduling in the ``flow.conf`` file.
```

Comment:
Does this make it sound as if our config names are confusing?

Reply:
Also, should it be...? 
```suggestion
        This should not be confused with scheduling in the ``flow.cylc`` file.
```